cmd/tether/tether.go

```diff
@@ -233,9 +241,20 @@ func launch(session *SessionConfig) error {
 	session.Cmd.Stderr = session.errwriter
 	session.Cmd.Stdin = session.reader
 
+	resolved, err := exec.LookPath(session.Cmd.Path)
+	if err != nil {
+		pretty := fmt.Errorf("%s: no such file or directory", session.Cmd.Path)
+		log.Errorf("Path lookup failed for %s: %s", session.Cmd.Path, err)
+		session.Started = pretty.Error()
+		return err
```

Comment:
Did you want to return `pretty` here instead of `err`?  Otherwise, pretty could just be a string (fmt.Sprintf intead of fmt.Errorf)